src/features/chat/components/leftpanel/StorageContent.tsx

```
@@ -22,29 +24,39 @@ interface StorageContentProps {
-    { id: 4, label: "더미 파일 3", type: "ai", ocrStatus: "completed" },
-    { id: 5, label: "더미 파일 4", type: "ai", ocrStatus: "completed" },
-  ]);
-
-  const [threadFiles, setThreadFiles] = useState<MockFile[]>([
-    { id: 100, label: "THREAD 1번", type: "ai", ocrStatus: "completed" },
-  ]);
+      fileUrl: asset.thumbnailUrl ?? undefined,
+      mimeType: asset.fileType, // fileType이 mimeType 형태라고 가정 (e.g. application/pdf)
+      ocrStatus: asset.ocrStatus as StorageFile["ocrStatus"],
+    }));
+  }, [noteDetail?.assets]);
```

ESLint

[error] Compilation Skipped: Existing memoization could not be preserved

React Compiler has skipped optimizing this component because the existing manual memoization could not be preserved. The inferred dependencies did not match the manually specified dependencies, which could cause the value to change more or less frequently than expected. The inferred dependency was `noteDetail.assets`, but the source dependencies were [noteDetail?.assets]. Inferred different dependency than source.

/home/jailuser/git/src/features/chat/components/leftpanel/StorageContent.tsx:34:43
  32 |
  33 |   // BOX 파일 목록 (업로드된 자산)
> 34 |   const boxFiles = useMemo<StorageFile[]>(() => {
     |                                           ^^^^^^^
> 35 |     if (!noteDetail?.assets) return [];
     | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> 36 |     return noteDetail.assets.map((asset) => ({
     …
     | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> 43 |     }));
     | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> 44 |   }, [noteDetail?.assets]);
     | ^^^^ Could not preserve existing manual memoization
  45 |
  46 |   // THREAD 파일 목록 (AI 생성 파일)
  47 |   const threadFiles = useMemo<StorageFile[]>(() => {

(react-hooks/preserve-manual-memoization)